Data/Sys/GameSettings/SD2J01.ini

```diff
@@ -26,8 +26,8 @@ PH_ZNear =
 PH_ZFar =
 
 [Video_Settings]
-SafeTextureCacheColorSamples = 512
```

Comment:
I just tested with the latest dev, the intro videos when you boot the game are erratic here.